components/k8s-model-server/images/releaser/components/workflows.libsonnet

```diff
@@ -111,21 +187,30 @@
                   "IMAGE=" + serving_image + "-${JOB_TYPE}-${PULL_BASE_SHA};" +
                   "until docker ps; do sleep 3; done; " +
                   "docker build --pull -t ${IMAGE} " +
-                  srcDir + "/components/k8s-model-server/docker/; " +
-                  "gcloud docker -- push ${IMAGE}",
+                      srcRootDir + "/kubeflow/kubeflow/components/k8s-model-server/docker/; " +
+                  "gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}; "  +
+                  "gcloud docker -- push ${IMAGE}"
                 ],
                 env: [
                   {
                     name: "DOCKER_HOST",
                     value: "127.0.0.1",
                   },
+                  {
+                    name: "GOOGLE_APPLICATION_CREDENTIALS",
```

Comment:
Why do you need to add this? Can't you use the BUILD_TEMPLATE and get this environment credential set by default? Then you just need to add extra variables?